src/mpid/ch4/src/ch4_progress.h

```diff
@@ -13,6 +13,8 @@
 
 #include "ch4_impl.h"
 
+#define MAX_PROGRESS_ATTEMPTS 50
```

Comment:
Namespace?

Reply:
Would `MPIDI_MAX_PROGRESS_ATTEMPTS` be correct?